static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,70 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project? (As an
+[external dependency](/doc/user-guide/external-dependencies))
+
+One way is to download the data (with `wget` or `dvc get`, for example) and use
+`dvc add` to track it locally, but the connection between projects would be
+lost: we couldn't tell where the data came from or whether there are new
+versions available. We offer the `dvc import` command instead (for projects
+hosted on a Git repository). It downloads the data, and creates a
+[DVC-file](/doc/user-guide/dvc-file-format) to track changes in the imported
+data.
+
+<details>
+
+### Expand to for an example
```

Comment:
`to for` typo? to lear more?

Reply:
Ugh yes. **~~to~~ for**. Fixing, thanks.